### PR TITLE
Some cleanup and improvements

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

**Comment by lburgazzoli** [comment]
ok to test

File: extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelBeanBuildItem.java [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 import java.util.Objects;
 
+import javax.annotation.Nullable;
```
Comment (ppalaga) [inline extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelBeanBuildItem.java]: I never used jsr305 and my naive expectation is that findbugs/spotbugs is requited to enforce it. Or it works differently? 

jsr305 artifact currently being a transitive dep of guava suggests to be cautious. Can't this potentially become a maintenance issue?

File: tooling/eclipse-formatter-config.xml [hunk 2]
```diff
@@ -267,5 +267,6 @@
         <setting id="org.eclipse.jdt.core.formatter.enabling_tag" value="CHECKSTYLE:ON"/>
         <setting id="org.eclipse.jdt.core.formatter.comment.format_block_comments" value="false"/>
         <setting id="org.eclipse.jdt.core.formatter.comment.format_line_comments" value="false"/>
+        <setting id="org.eclipse.jdt.core.formatter.comment.format_javadoc_comments" value="false"/>
```
Comment (ppalaga) [inline tooling/eclipse-formatter-config.xml]: What problem is this solving?

File: extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelBeanBuildItem.java (hunk 1 above)
Comment (lburgazzoli) [reply extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelBeanBuildItem.java]: it is primary for documentation purpose here

File: tooling/eclipse-formatter-config.xml (hunk 2 above)
Comment (lburgazzoli) [reply tooling/eclipse-formatter-config.xml]: The formatter plugin does not keep aliment with parameter which hurts readability

Comment (ppalaga) [reply tooling/eclipse-formatter-config.xml]: Could you please try if the following combination does what you want?

```
<setting id="org.eclipse.jdt.core.formatter.comment.format_javadoc_comments" value="true"/>
<setting id="org.eclipse.jdt.core.formatter.comment.align_tags_names_descriptions" value="true"/>
<setting id="org.eclipse.jdt.core.formatter.comment.indent_root_tags" value="false"/>
<setting id="org.eclipse.jdt.core.formatter.comment.align_tags_descriptions_grouped" value="false"/>
```

Comment (lburgazzoli) [reply tooling/eclipse-formatter-config.xml]: would you mind sending a follow up pr ? I don't want to waste another build cycle and I'm getting a little tired of this auto formatter thing

**Pull request merged**